stores/moderationScreenStore.ts

```diff
@@ -18,6 +18,13 @@ export enum ModSubmissionLeftNavbarSectionIDs {
     HealthcareProfessionalName = 'HEALTHCARE_PROFESSIONAL_NAME'
 }
 
+export enum ModHealthcareProfessionalsLeftNavbarSectionIDs {
```

Comment:
```suggestion
export enum ModHealthcareProfessionalLeftNavbarSections {
```
I think we can simplify the name a bit since it's quite lengthy.